bin/skel.py

```diff
@@ -116,7 +116,7 @@ def new_problem():
         validation = _ask_variable('validation (default/custom/custom interactive)', 'default')
 
     # Read settings from the contest-level yaml file.
-    variables = read_yaml_settings(Path('contest.yaml'))
+    variables = contest.contest_yaml() or {}
```

Comment:
Maybe `contest_yaml()` should just return `{}` instead of `None` when the file isn't found

Reply:
Yeah, I was reluctant to do this, because some usages of `contest_yaml()` have different behaviour when there is no contest config. But I can take a look at this to see if they can be rewritten to simply check for `{}` instead (good thing that `bool({}) == False` in Python :joy:)